sha2/README.md

```diff
@@ -7,12 +7,52 @@
 [![Project Chat][chat-image]][chat-link]
 [![Build Status][build-image]][build-link]
 
-Pure Rust implementation of the [SHA-2 hash function family][1]
-including SHA-224, SHA-256, SHA-384, and SHA-512.
+Pure Rust implementation of the [SHA-2] cryptographic hash algorithms.
 
-[Documentation][docs-link]
+There are 6 standard algorithms specified in the SHA-2 standard: 
+[`Sha224`], [`Sha256`], [`Sha512_224`], [`Sha512_256`], [`Sha384`], and [`Sha512`].
 
-<img src="https://raw.githubusercontent.com/RustCrypto/meta/master/img/hashes/sha2.png" width="480px">
```

Comment:
Aww 😄

Reply:
I think most users of the crate (and thus readers of the doc) would not care for this pretty picture. :)

Reply:
I can add it after examples if you want.

Reply:
Whatever